spark/src/main/scala/filodb.spark/FiloRelation.scala

```diff
@@ -92,6 +95,15 @@ object FiloRelation extends StrictLogging {
     }
   }
 
+  /**
+    * Method to get possible combinations of key values provided in partition key filters.
+    * For example filters provided in the query are actor2Code IN ('JPN', 'KHM') and year = 1979
+    * then this method returns all combinations like ('JPN',1979) , ('KHM',1979) which represents
+    * full partition key value.
+    * @param xs
```

Comment:
I think remove both.  Thanks.

> On Jul 6, 2016, at 6:24 AM, parekuti <notifications@github.com> wrote:
> 
> In spark/src/main/scala/filodb.spark/FiloRelation.scala <https://github.com/tuplejump/FiloDB/pull/110#discussion_r69728599>:
> 
> > @@ -92,6 +95,15 @@ object FiloRelation extends StrictLogging {
> >      }
> >    }
> >  
> > +  /**
> > +    * Method to get possible combinations of key values provided in partition key filters.
> > +    * For example filters provided in the query are actor2Code IN ('JPN', 'KHM') and year = 1979
> > +    * then this method returns all combinations like ('JPN',1979) , ('KHM',1979) which represents
> > +    * full partition key value.
> > +    * @param xs
> Should i just remove @param <https://github.com/param> only or @tparam A @return <https://github.com/return> as well.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/tuplejump/FiloDB/pull/110/files/706f13689d820bbd9dff9a21375aa1a6c06478d6..6cd8c64159157c142a5f94994af5a81bf460f4d0#r69728599>, or mute the thread <https://github.com/notifications/unsubscribe/ABA327Uo6nck6vJ_VUD7PHTYOyIKYFWpks5qS6yqgaJpZM4JCOww>.
>